ext/pdo_odbc/tests/long_columns.phpt

```diff
@@ -92,6 +92,12 @@ foreach ($db->query('SELECT id, data from TEST ORDER BY LEN(data) ASC') as $row)
 
 echo "Finished\n";
 ?>
+--CLEAN--
+<?php
+require 'ext/pdo/tests/pdo_test.inc';
+$db = PDOTest::test_factory(dirname(__FILE__) . '/common.phpt');
+$db->exec("DROP TABLE IF EXISTS test_long_columns");
```

Comment:
Use new method?

Reply:
I'm not sure 🤔  The method is needed when the syntax is not known in advance, otherwise it is just noise

Reply:
That's a fair point.